plugins/woocommerce/templates/order/order-details-fulfillments.php

```diff
@@ -0,0 +1,257 @@
+<?php
+/**
+ * Order details
+ *
+ * This template can be overridden by copying it to yourtheme/woocommerce/order/order-details.php.
```

Comment:
The path needs to be updated, and in the other template too 🙂

Reply:
Fixed in [bc9d95b](https://github.com/woocommerce/woocommerce/pull/58713/commits/bc9d95bec39d60718ea6f54582b4f3a366ca8c23)